presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -32,6 +35,8 @@
     private boolean caseInsensitiveNameMatching;
     private Duration caseInsensitiveNameMatchingCacheTtl = new Duration(1, MINUTES);
     private Set<String> jdbcTypesMappedToVarchar = ImmutableSet.of();
+    private int decimalDefaultScale = 16;
```

Comment:
I am not sure about this default. Maybe 0 would be a better default. Alternatively we could leave it unset and do not map unscalled decimals with this property is not configured so user is forced to pass a good default on his own.

WDYT?

@findepi Any thoughts?

Reply:
i would choose 9, but this is an arbitrary choice. 16 looks good to me.

Reply:
I found `16` as a suitable compromise between not loosing actual source scale values and decreasing possible precision.

A new boolean toggle can be added (`decimal-rounding-enable`) to let users decide between keeping current skipping behaviour or using rounding with the risk of failing on decimal overflow.